src/main/java/it/gov/pagopa/pu/fileshare/exception/custom/FlowFileNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package it.gov.pagopa.pu.fileshare.exception.custom;
+
+public class FlowFileNotFoundException extends RuntimeException {
```

Comment:
what's a FlowFile? :D 
```suggestion
public class FileNotFoundException extends RuntimeException {
```

Reply:
ps: can you add it's throws also in ingestionFlowFile donwload?